sct_sizing.py

```
@@ -0,0 +1,589 @@
+        role_values: dict[str, object] = {}
+        role_sources: dict[str, str] = {}
+        agnostic_param = AGNOSTIC_PARAMS[role]
+        agnostic_value = merged_config.get(agnostic_param)
+        for cloud_name in clouds:
+            param_name = CLOUD_PARAM[cloud_name].get(role)
+            if not param_name:
+                continue
+            if param_name in cli_overrides and cli_overrides[param_name]:
+                role_values[cloud_name] = cli_overrides[param_name]
+                role_sources[cloud_name] = "cli"
+            elif agnostic_param in cli_overrides and cli_overrides[agnostic_param]:
```

Ruff (0.15.14)

[warning] Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)